crates/async-sawtooth-sdk/src/error.rs

```diff
@@ -0,0 +1,45 @@
+use futures::channel::mpsc::SendError;
+use protobuf::ProtobufError;
+use sawtooth_sdk::messaging::stream::ReceiveError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum SawtoothCommunicationError {
+    #[error("ZMQ error {0}")]
```

Comment:
```suggestion
    #[error("ZMQ error: {0}")]
```
If touching this anyway, some colons help with readability.

Reply:
Good spot

Reply:
Couple of other similar touched in other files, we'll get them all in the end though!

Reply:
Maybe the others in the same block at least?